internal/devicescale/impl_desktop_notx.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Ebiten Authors
+// Copyright 2021 The Ebiten Authors
```

Comment:
Please rename this file to `impl_notx.go`

(The name doesn't reflect the exact situation but I'd like to keep the filename convention `foo_bar.go`).

Reply:
Done.